src/power_grid_model_io/converters/pandapower_converter.py

```diff
@@ -35,7 +35,7 @@
 
 
 # pylint: disable=too-many-instance-attributes
-class PandaPowerConverter(BaseConverter[PandaPowerData]):
+class PandaPowerConverter(BaseConverter[PandaPowerData]):  # pragma: no cover
```

Comment:
Does this ignore the entire class? Then the coverage gets ignored for all versions too right?

Reply:
let's just hope that we do not need to wait so long before pp fixes their stuff